spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/query/QuerySpec.java

```diff
@@ -67,7 +67,7 @@ public SortOrder getSortOrder() {
     }
 
     public static final class Builder {
-        private static final String EQUALS_EXPRESSION_PATTERN = "[^\\s\\\\]*(\\s*)=(\\s*)[^\\s\\\\]*";
+        private static final String EQUALS_EXPRESSION_PATTERN = "[^\\s\\\\]*(\\s*)=(\\s*)[^\\\\]*";
```

Comment:
this regex matches the same expressions that a `contains("=")` will do, I will suggest to avoid the regex then

Reply:
not quite. "some key=some value" (left hand contains whitespace) would match `contains("=")` but not the regex